package.json

```diff
@@ -32,6 +32,7 @@
     "@actions/io": "^1.1.1",
     "ansi-colors": "^4.1.1",
     "axios": "1.1.2",
+    "github-emoji": "^1.2.0",
```

Comment:
Since this one is not needed in the end we should remove it from both here and the `package-lock.json`